### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by ShowBaba** [Open-Currency-Collective/nubian-dsa-connect#3]
I tried using the package and I keep getting back this error: 
**SyntaxError: Cannot use import statement outside a module**

```
-- trace: /node_modules/nubian-dsa-connect/dist/index.js:12
import e from"axios";var t=function(e,n){return(t=Object.setPrototypeOf||{__proto__:[]}instanceof Array&&function(e,t){e.__proto__=t}||function(e,t){for(var n 
```

I tried adding `"type": "module",` to package.json and change to `import DSA from 'nubian-dsa-connect'` instead of _require_ 
that didn't solve it.

Any help please

**Comment by EbubeUd** [comment]
Currently working on this issue. It should be resolved in the next update to the SDK.

**Comment by EbubeUd** [comment]
This issue has been resolved in version 0.4.1
